datasetinsights/estimators/faster_rcnn.py

```diff
@@ -33,44 +36,102 @@ class FasterRCNN(Estimator):
     """
     Faster-RCNN train/evaluate implementation for object detection.
 
-    https://github.com/pytorch/vision/tree/master/references/detection
-    https://arxiv.org/abs/1506.01497
+
     Args:
         config (CfgNode): estimator config
-        writer: Tensorboard writer object
-        kfp_writer: KubeflowPipelineWriter object to write logs
-        checkpointer: Model checkpointer callback to save models
-        device: model training on device (cpu|cuda)
         box_score_thresh: (optional) default threshold is 0.05
-        gpu (int): (optional) gpu id on which code will execute
-        rank (int): (optional) rank of process executing code
+        distributed: whether or not the estimator is distributed
+        data_root: Root directory on localhost where datasets are located
```

Comment:
Actually when the estimator is initialized, it does not care the data_root parameter as no data are loaded in this stage. I think it's better to have data_root parsed at later stage when `estimator.train(...)` is called.

Reply:
I agree i just did not want to keep refactoring.